scripts/trello/api.js

```diff
@@ -0,0 +1,19 @@
+async function get(relativeUrl, params, { trelloApiKey, trelloOAuth1 } = {}) {
```

Comment:
@AdriAt360 get / post / put are becoming globals. I know the project will keep small, but it's possibly already too dirty.

 - Could be interesting to at least "tidy" them in a TrelloApi to use them like `TrelloClient.get(...)`.
 - Other idea: keep key / token in an instance contex like this, which will simplify the context handling in your project
```javascript
const trelloClient = new TrelloClient(key, token);
trelloClient.get(** less_infos here :) **);
```

Reply:
@pp0rtal good idea to introduce a `TrelloClient` here! https://github.com/360Learning/360learning.github.io/pull/10/commits/c480d98c0f9f102eb3e351e63eef76b513b7a61c